recipes/approvaltests.cpp/8.0.0/conandata.yml

```diff
@@ -0,0 +1,6 @@
+sources:
+  8.0.0:
+    - url: https://github.com/approvals/ApprovalTests.cpp/releases/download/v.8.0.0/ApprovalTests.v.8.0.0.hpp
+      sha256: e16a97081f8582be951d95a9d53dc611f1f5a84e117a477029890d0b34ae99d6
+    - url: "https://raw.githubusercontent.com/approvals/ApprovalTests.cpp/v.8.0.0/LICENSE"
+      sha256: c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4
```

Comment:
You can add `8.1.0` too if you want to. It is the [latest release available](https://github.com/approvals/ApprovalTests.cpp/releases/tag/v.8.1.0)

Reply:
Done. Thanks @uilianries